$MirOS: ports/www/dillo/stable/patches/patch-src_cookies_c,v 1.2 2005/06/19 17:20:19 bsiegert Exp $
$OpenBSD: patch-src_cookies_c,v 1.3 2004/02/21 01:37:19 couderc Exp $
--- src/cookies.c.orig	Wed Mar 29 18:47:50 2006
+++ src/cookies.c	Mon May 15 17:50:36 2006
@@ -15,7 +15,6 @@
  * http://www.cis.ohio-state.edu/cs/Services/rfc/rfc-text/rfc2965.txt
  */
 
-#define DEBUG_LEVEL 10
 #include "debug.h"
 
 
@@ -229,7 +228,7 @@ static int Cookie_control_init(void)
    gboolean enabled = FALSE;
 
    /* Get a file pointer */
-   filename = a_Misc_prepend_user_home(".dillo/cookiesrc");
+   filename = a_Misc_prepend_user_home(".etc/dillo/cookiesrc");
    stream = Cookies_fopen(filename, "DEFAULT DENY\n");
    g_free(filename);
 
